dash.js

```diff
@@ -77,6 +78,18 @@ function reset() {
     dash._queueRedisplay();
 }
 
+function apps_to_top() {
+    let dash = Main.overview._overview._controls.dash;
+    dash._dashContainer.remove_child(dash._showAppsIcon);
+    dash._dashContainer.insert_child_at_index(dash._showAppsIcon, 0);
```

Comment:
This will move the child if it exists:
```suggestion
    dash._dashContainer.set_child_at_index(dash._showAppsIcon,0);
```